### ISSUE TITLE
Add 'maintain aspect ratio' modifier keybind for selection resize while dragging.

### ISSUE THREAD

**Issue by Flightkick** [PintaProject/Pinta#586]
**Description**
AFAIK, there's no modifier to keep the aspect ratio locked while resizing a selected area using mouse dragging.
It would be nice if this was possible, this makes it easier to align layers without distorting ratios.

In Paint.NET this is achieved by holding down the `Shift` key while dragging.

**Comment by cameronwhite** [comment]
For the Move Selected Pixels tool, holding Ctrl + Shift should keep the aspect ratio
For the rectangle select tool, Shift only constrains to a square so this would be something to keep in mind when making the changes for #586

**Comment by Flightkick** [comment]
@cameronwhite Ah thanks, glad this is supported. Though as Pinta seems to be heavily inspired by Paint.NET, is there any reason for the discrepancy between keybinds (`Shift` v.s. `Ctrl` + `Shift`)?

Also, I haven't been able to find any documentation regarding this behavior, which may be nice to add.

Thanks again!

**Comment by cameronwhite** [comment]
> @cameronwhite Ah thanks, glad this is supported. Though as Pinta seems to be heavily inspired by Paint.NET, is there any reason for the discrepancy between keybinds (`Shift` v.s. `Ctrl` + `Shift`)?
I don't think it was really intended to be different, but Ctrl is used to activate scaling in the Move Selected Pixels tool because it doesn't have any handles to use for scaling (as suggested in #585)

> Also, I haven't been able to find any documentation regarding this behavior, which may be nice to add.
The hotkeys for a particular tool are in the tooltip if you hover over the tool in the tools panel, but the Move Selected Pixels tool doesn't seem to mention using Shift to constrain the scaling
